api/tests/unit/external_resources/test_unit_external_resources_views.py

```diff
@@ -0,0 +1,136 @@
+import simplejson as json
+from django.core.serializers.json import DjangoJSONEncoder
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from features.feature_external_resources.models import FeatureExternalResource
+from features.models import Feature
+
+_django_json_encoder_default = DjangoJSONEncoder().default
+
+
+def test_create_external_resources(
+    admin_client: APIClient,
```

Comment:
Can we add `master_api_key_client` to all these tests as well?

Reply:
Changed